sat/cli/bootsys/parser.py

```diff
@@ -55,7 +55,7 @@
                 'compute and application nodes have completed their BOS shutdown.'),
     TimeoutSpec('bos-boot', ['boot', 'reboot'], 900,
                 'compute and application nodes have completed their BOS boot.'),
-    TimeoutSpec('ncn-shutdown', ['shutdown'], 300,
+    TimeoutSpec('ncn-shutdown', ['shutdown'], 900,
```

Comment:
I forgot that we also specify this default value in the man page. Unfortunately, the usage info and man pages have separate source.

See line 250 of the file `docs/man/sat-bootsys.8.rst`, and please update the default value there.